Makefile

```diff
@@ -1,26 +1,30 @@
-CFLAGS=		-g -Wall -O2 -Wc++-compat #-Wextra
+CFLAGS=		-g -Wall -O2 -Wc++-compat  #-Wextra
 CPPFLAGS=	-DHAVE_KALLOC
-INCLUDES=
-OBJS=		kthread.o kalloc.o misc.o bseq.o sketch.o sdust.o options.o index.o chain.o align.o hit.o map.o format.o pe.o esterr.o splitidx.o ksw2_ll_sse.o
+INCLUDES=   -Ilib/simde/simde
```

Comment:
This could be `-Ilib/simde` rather than `-Ilib/simde/simde`, modifying the load header paths. Because for example `#include <simde/x86/sse2.h>` is the right call by the simde system library (such as `libsimde-dev` in Debian) or `simde-devel` in Fedora and EPEL (CentOS).

https://wiki.debian.org/SIMDEverywhere

Reply:
I've changed to this in the following commit.